### PR TITLE
Renderer docs !

### PR DESCRIPTION
**Pull request by alvinsight**

### CODE REVIEW

File: src/core/renderers/webgl/filters/SpriteMaskFilter.js [hunk 1]
```diff
@@ -72,6 +70,13 @@ SpriteMaskFilter.prototype = Object.create(AbstractFilter.prototype);
 SpriteMaskFilter.prototype.constructor = SpriteMaskFilter;
 module.exports = SpriteMaskFilter;
 
+/**
+ * Applies the filter ? @alvin
+ *
+ * @param renderer {WebGLRenderer} A reference to the WebGL renderer
+ * @param input {}
+ * @param output {}
```
Comment (englercj) [inline src/core/renderers/webgl/filters/SpriteMaskFilter.js]: input/output should be RenderTargets.

File: src/core/renderers/webgl/managers/FilterManager.js [hunk 2]
```diff
@@ -205,11 +216,22 @@ FilterManager.prototype.getRenderTarget = function ( clear )
...
 FilterManager.prototype.returnRenderTarget = function (renderTarget)
 {
     this.texturePool.push( renderTarget );
 };
 
+/* 
+ * Applies the filter
+ * @param shader {Shader} 
+ * @param inputTarget {Any}
+ * @param outputTarget {Any}
```
Comment (englercj) [inline src/core/renderers/webgl/managers/FilterManager.js]: these are RenderTargets IIRC

File: src/core/renderers/webgl/filters/SpriteMaskFilter.js (hunk 1 above)
Comment (alvinsight) [inline src/core/renderers/webgl/filters/SpriteMaskFilter.js]: I just changed it, thanks man :)

**Comment by GoodBoyDigital** [comment]
:+1:

**Pull request merged**